projects/canopy/src/lib/modal/modal.component.spec.ts

```diff
@@ -60,44 +60,60 @@ describe('LgModalComponent', () => {
   });
 
   describe('on init', () => {
-    it('should update isOpen', () => {
-      isModalOpen$.next(true);
+    describe('when the modal hasn\'t been opened/closed yet', () => {
+      it('should update isOpen', () => {
```

Comment:
should this text change to match the test?

Reply:
What do you mean?

Reply:
`should update isOpen to undefined` i guess?